src/components/splash/Splash.js

```diff
@@ -56,7 +56,7 @@ const Splash = ({ animation, isLoggedIn }) => {
   }
 
   return (
-    <Wrapper style={styles.wrapper}>
+    <Wrapper style={isMobileNative ? styles.mobileWrapper : styles.wrapper}>
```

Comment:
i've noticed this also on my mobile when i use the app as installed webapp not using the native app.
i think isMobileNative will  only fix it on the react-native version